ruby/tests/common_tests.rb

```diff
@@ -1119,4 +1119,43 @@ def test_json_emit_defaults_repeated_submsg
   def test_comparison_with_arbitrary_object
     assert proto_module::TestMessage.new != nil
   end
+
+  def test_converts_time
+    m = proto_module::TimeMessage.new
+
+    m.timestamp = Google::Protobuf::Timestamp.new(seconds: 5, nanos: 6)
```

Comment:
Might be nice to add a `assert_kind_of` assertion here for `m.timestamp`, really just to be explicit and for documentation.

Reply:
Sure, I'll add it.